src/o365/flow/commands/flow-remove.ts

```diff
@@ -0,0 +1,171 @@
+        url: `${this.resource}providers/Microsoft.ProcessSimple/${args.options.asAdmin ? 'scopes/admin/' : ''}environments/${encodeURIComponent(args.options.environment)}/flows/${encodeURIComponent(args.options.name)}?api-version=2016-11-01`,
+        resolveWithFullResponse: true,
+        headers: {
+          accept: 'application/json'
+        },
+        json: true
+      };
+
+      request
+        .delete(requestOptions)
+        .then((rawRes: any): void => {
+          // handle 204 and throw error message to cmd when invalid flow id is passed
+          // https://github.com/pnp/office365-cli/issues/1063#issuecomment-537218957
+          if (rawRes.statusCode === 204) {
+            cmd.log(vorpal.chalk.red(`Error: Resource '${args.options.name}' does not exist in environment '${args.options.environment}'`));
+            cb();
+          } else {
+            if (this.verbose) {
+              cmd.log(vorpal.chalk.green('DONE'));
+            }
+            cb();
+          }
+        }, (rawRes: any): void => this.handleRejectedODataJsonPromise(rawRes, cmd, cb));
+    };
+    if (args.options.confirm) {
+      removeFlow();
+    }
+    else {
+      cmd.prompt({
+        type: 'confirm',
+        name: 'continue',
+        default: false,
+        message: `Are you sure you want to remove the Microsoft Flow ${args.options.name}?`
+      }, (result: { continue: boolean }): void => {
+        if (!result.continue) {
+          cb();
+        }
+        else {
+          removeFlow();
+        }
+      });
+    }
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-n, --name <name>',
+        description: 'The name of the Microsoft Flow to remove'
+      },
+      {
+        option: '-e, --environment <environment>',
+        description: 'The name of the environment for which to remove flow'
+      },
+      {
+        option: '--asAdmin',
+        description: 'Set, to remove the Flow as admin'
+      },
+      {
+        option: '--confirm',
+        description: 'Don\'t prompt for confirmation'
+      },
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(): CommandValidate {
+    return (args: CommandArgs): boolean | string => {
+      if (!args.options.name) {
+        return 'Required option name missing';
+      }
+
+      if (!Utils.isValidGuid(args.options.name)) {
+        return `${args.options.name} is not a valid GUID`;
+      }
+
+      if (!args.options.environment) {
+        return 'Required option environment missing';
+      }
+
+      return true;
+    };
+  }
+
+  public commandHelp(args: {}, log: (help: string) => void): void {
+    const chalk = vorpal.chalk;
+    log(vorpal.find(commands.FLOW_REMOVE).helpInformation());
+    log(
+      `  Remarks:
+
+    ${chalk.yellow('Attention:')} This command is based on an API that is currently
```

Comment:
Is this really the case?

Reply:
I left this in for consistency as all the other flow commands have this same remark